### PR TITLE
ADO#4864 -- Status title; Documents must have <title> element to aid in navigation

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
### Description

List of proposed changes:

- Added in dynamic loaded of status titles based on the status code received (same title as header titles)

### CODE REVIEW

File: src/pages/status.tsx [hunk 1]
```diff
@@ -164,13 +164,32 @@ const Status = () => {
     router.push('/landing')
   }, [router])
 
+  function getTitleHeader(code: string | undefined): string {
```
Comment (ken-blanchard) [inline src/pages/status.tsx:167]: This does not account for "response-no-result" scenario. Difference between checkStatusResponse being undefined or null.

image

Comment (AdamP-HRSDC-RHDCC) [reply src/pages/status.tsx]: Updated the cases to include null

Comment (ken-blanchard) [reply src/pages/status.tsx]: Updated version still isn't correct.

 switch (checkStatusResponse?.status) will evaluate to undefined in both null|undefined cases.
 
 Instead, early return on undefined.
 
if( checkStatusResponse === undefined) return t('header')

// else do the switch.

Comment (AdamP-HRSDC-RHDCC) [reply src/pages/status.tsx]: Added in conditional to return for only undefined

**Review by ken-blanchard — APPROVED** [review]

**Pull request merged**